### ISSUE TITLE
Missing title on iframe element template

### ISSUE THREAD

**Issue by tardinha** [nswdpc/waratah#35]
[Missing title](https://github.com/nswdpc/waratah/blob/f4965fe45fc53ad387031d66b61ae28f04d52d70/themes/nswds/templates/NSWDPC/Elemental/Models/Iframe/ElementIframe.ss) on iframe template.

See: [WCAG 2.1 A 2.4.1 Section 508 (2017) A 2.4.1](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H64)

**Comment by JamesDPC** [comment]
Is `AlternateContent` not working ?

**Comment by tardinha** [comment]
Ahh... AlternateContent populates the title, not the iframe title.

**Comment by JamesDPC** [comment]
For WCAG conformance, the AlternateContent field should be required.

Apart from a typo, the description of the field is OK:
https://github.com/nswdpc/silverstripe-elemental-iframe/blob/master/src/Models/Elements/ElementIframe.php#L451
